tests/libraries/test_arrays.py

```diff
@@ -1,55 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Why do we delete these tests? They're testing the uint256 array library which is still being used I think

Reply:
What's using the uint256 array library?  The only place I saw it being used was ArrayHelper, which was only being used by test_arrays.py.

Reply:
not used I guess, just not deleted. As long as we're keeping it I'd prefer it stay tested.